uprotocol_vsomeip/examples/publish_subscribe.py

```diff
@@ -0,0 +1,153 @@
+# -------------------------------------------------------------------------
+#
+# Copyright (c) 2024 General Motors GTO LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# SPDX-FileType: SOURCE
+# SPDX-FileCopyrightText: 2024 General Motors GTO LLC
+# SPDX-License-Identifier: Apache-2.0
+#
+# -------------------------------------------------------------------------
+"""
+Publish Subscribe Example
+"""
+import logging
+import time
+import socket
+from typing import List
+
+from google.protobuf import any_pb2
+from uprotocol.transport.ulistener import UListener
+from uprotocol.proto.uri_pb2 import UUri, UAuthority, UEntity, UResource
+from uprotocol.proto.umessage_pb2 import UMessage
+from uprotocol.proto.upayload_pb2 import UPayloadFormat, UPayload
+from uprotocol.proto.uattributes_pb2 import UPriority
+from uprotocol.transport.builder.uattributesbuilder import UAttributesBuilder
+from uprotocol_vsomeip.vsomeip_utransport import VsomeipTransport, VsomeipHelper
+from target.protofiles.ultifi.vehicle.body.cabin_climate.v1 import (
```

Comment:
Remove all usage of ultifi protos from everywhere

Reply:
Noted and fixed to use generic values instead of ultifi protos